front/src/config/i18n/en.json

```diff
@@ -791,6 +791,18 @@
         "saveLabel": "Save configuration"
       }
     },
+    "oauth2": {
+      "buttonConnect": "Connect now",
+      "apiKeyLabel": "Then, enter your client_id and secret_id key here:",
+      "errorEmptyConfig": "Client Id and Secret Id must not be empty !",
+      "errorServiceId": "Service Id of current integration must not be found !",
+      "errorAuthorizationUri": "Error in authorization provider !",
+      "clientId": "Account client id",
+      "secretId": "Account secret id",
+      "instructions": "In this stage please fill in the given client_id and secret_id key below and click on connect button.",
```

Comment:
Not sure `client_id` is user friendly, maybe `client id` is better.
Client secret instead of secret_id.

Reply:
i fix it